CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.2.3 (December 26, 2019)
+    * Fix `Error! Cannot convert undefined or null to object` error on no attachments object in message in Write file action
+
+## General Changes
+    * Update sailor version to 2.5.4
```

Comment:
Cant see this changes in package.json. Also why not upgrading to sailor 2.6.3?

Reply:
Stupid copypaste mistake. Upgrading is out of scope